core/iwasm/libraries/lib-rats/lib_rats_wrapper.c

```diff
@@ -104,7 +113,8 @@ librats_parse_evidence_wrapper(wasm_exec_env_t exec_env,
 static NativeSymbol native_symbols_lib_rats[] = {
     REG_NATIVE_FUNC(librats_collect, "(**~)i"),
     REG_NATIVE_FUNC(librats_verify, "(*~*~)i"),
-    REG_NATIVE_FUNC(librats_parse_evidence, "(*~*~)i")
+    REG_NATIVE_FUNC(librats_parse_evidence, "(*~*~)i"),
+    REG_NATIVE_FUNC(librats_dispose_evidence_json, "(*)")
```

Comment:
Seems it should be "(i)", here the evidence_json returned from librats_parse_evidence is already uint32 offset in wasm world, which can be freed by module_free, no need to convert to pointer again by runtime, see librats_collect_wrapper:
```C
str_ret_offset = module_malloc(json_size, (void **)&str_ret);
...
*((int *)evidence_json) = str_ret_offset;
```

BTW, had better declare librats_collect_wrapper as
`... librats_collect_wrapper (..., uint32_t *evidence_json, ...)`, and directly set evidence_json:
```C
*evidence_json = str_ret_offset;
```